package.json

```diff
@@ -33,42 +33,37 @@
   },
   "namespace": "@tanstack",
   "devDependencies": {
-    "@commitlint/parse": "^17.6.5",
-    "@solidjs/testing-library": "^0.8.4",
-    "@testing-library/jest-dom": "^5.16.5",
-    "@testing-library/react": "^14.0.0",
+    "@commitlint/parse": "^18.4.3",
+    "@solidjs/testing-library": "^0.8.5",
+    "@testing-library/jest-dom": "^6.1.5",
+    "@testing-library/react": "^14.1.2",
     "@testing-library/react-hooks": "^8.0.1",
     "@testing-library/user-event": "^14.4.3",
     "@testing-library/vue": "^7.0.0",
     "@types/current-git-branch": "^1.1.6",
-    "@types/git-log-parser": "^1.2.3",
     "@types/eslint": "^8.56.0",
+    "@types/git-log-parser": "^1.2.3",
     "@types/jest": "^26.0.4",
     "@types/jsonfile": "^6.1.4",
     "@types/luxon": "^2.3.1",
-    "@types/node": "^18.15.3",
+    "@types/node": "^18.19.3",
     "@types/react": "^18.0.14",
     "@types/react-dom": "^18.0.5",
-    "@types/semver": "^7.3.13",
+    "@types/semver": "^7.5.6",
     "@types/stream-to-array": "^2.3.3",
-    "@types/testing-library__jest-dom": "^5.14.5",
```

Comment:
Wait, this isn't needed? I thought our tests utilized this

Reply:
@testing-library/jest-dom v6 includes their own types :)